utils/graph.c

```diff
@@ -87,6 +88,7 @@ static int add_graph_entry(struct uftrace_task_graph *tg, char *name, size_t nod
 
 		node = xzalloc(node_size);
 
+		node->id = next_id++;
```

Comment:
Although this is a small change, I think it's better to split it as a separate commit because it's a general change.  Please add a comment to `graph_add_node()` function that it's not thread-safe due to the static id.

Reply:
Thank you for the review!
I added the comment for it to line 210 because graph_add_node is a public interface for others.
Do you think it is better to move it here?
Except it, I updated the code.

Reply:
> > Please add a comment to graph_add_node() function that it's not thread-safe due to the static id.
> 
> Please address @namhyung's comment above.

Thank you for the review!
I added the comment for utils/graph.c to line 210 because graph_add_node is a public interface for others.
Do you think it is better to move it here?
Except for it, I updated the code.